web/src/app/features/integrations/integrations.page.html

```diff
@@ -11,20 +11,50 @@ <h1 class="tr-title-1">Integrations</h1>
       (pageChange)="pageChanged($event)" />
   </div>
 
-  <div class="flex-grow bg-secondary rounded-md overflow-auto">
-    <div class="p-4 space-y-4 flex-col flex">
-      @if (loading()) { @for (_ of [].constructor(5); track $index) {
-      <div class="bg-white rounded-md p-4 flex gap-2">
-        <tr-skeleton class="w-12 h-12 rounded-full bg-secondary" />
-        <div class="space-y-2">
-          <tr-skeleton class="h-4 w-[250px] bg-secondary" />
-          <tr-skeleton class="h-4 w-[200px] bg-secondary" />
-        </div>
+  <div class="h-full bg-secondary rounded-md overflow-auto p-4">
+    @if (loading()) {
+      <div class="h-full flex flex-col gap-3">
+        @for (_ of [].constructor(5); track $index) {
+          <div class="bg-white rounded-md p-4 flex gap-2">
+            <tr-skeleton class="w-12 h-12 rounded-full bg-secondary" />
+            <div class="space-y-2">
+              <tr-skeleton class="h-4 w-[250px] bg-secondary" />
+              <tr-skeleton class="h-4 w-[200px] bg-secondary" />
+            </div>
+          </div>
+        }
       </div>
-      } } @if (integrations | async ; as page) { @if (!loading()) { @for
-      (integration of page.data; track integration.id) {
-      <tr-integration-linked-card [integration]="integration" />
-      } } }
-    </div>
+    }
+    @if (integrations | async; as page) {
+      @if (!loading() && page.data.length > 0) {
+        <div class="h-full flex flex-col gap-3">
+          @for (integration of page.data; track integration.id) {
+            <tr-integration-linked-card [integration]="integration" />
+          }
+        </div>
+      } @else {
+        <div class="h-full flex flex-col justify-center -mt-12">
```

Comment:
Please remove this useless margin. This move everything...
```suggestion
        <div class="h-full flex flex-col justify-center">
```